docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -272,7 +272,12 @@ question: |
   Information about people
 subquestion: |
   This form includes the following variables representing people:
-  ${ comma_and_list(set(people_variables.true_values()).union(set(built_in_people_vars_used))) }
+  ${ comma_and_list(person_candidates) }
+  
+  **Note**: 
+  If your form includes `defendants/respondents` or `plaintiffs/petitioners`,
+  those variables will be replaced with `users` and `other_parties` depending
+  on the role the user of the form has in the case.
```

Comment:
Maybe minor note: make this conditional on there actually being defendants/respondents in the form, which could be set as just a separate variable on line 556, and it could be really clear: "You form includes "defendants/respondents", etc.", instead of people wondering if their form actually has those things.